### PR TITLE
Happy Thoughts!

### PR DESCRIPTION
**Pull request by Josephineoderland**
## Netlify link

https://candid-begonia-4a8bea.netlify.app

### CODE REVIEW

**Review by dzc1 — APPROVED** [review]
Great work Josephine 👍  

The  structure of your project is great and advanced stretch requirements have been met.in all honesty, im loving seeing the logic structure within the topLevel file app.jsx of your project. II see that you are digging the use of ternary's!clean code is what we aim to teach here in Technigo so praise J! 

Now, I'll be honest, I will give you the approved grade given that all req's and advanced reqs have been met but, I want to challenge you in destructuring a bit more the htmL structure within your return statement of the app.jsx. If you feel commfortable, you can destructure these elements into components and pass the data as props.!

Great job nevertheless J! keep up the clean work!

File: src/App.jsx [hunk 1]
```diff
@@ -1,3 +1,211 @@
...
+                  </div>
+                </div>
+              </li>
+            ))}
+          </ul>
+        </>
+      )}
+    </div>
+  )
+}
```
Comment (Paulajungaker) [inline src/App.jsx:211]: Your code looks very good and clean and is easy to read. You could divide the code into more components to make it a bit more easy to find what part you are looking at, but very nice work:)

File: src/index.css [hunk 2]
```diff
@@ -11,3 +11,263 @@ code {
...
+  }
+
+  .show-liked-button {
+    font-size: 20px;
+  }
+
+  .like-button-container p {
+    font-size: 30px;
+  }
+}
```
Comment (Paulajungaker) [inline src/index.css:273]: The site looks very good and is very similar to the example we were to follow.

**Review by Paulajungaker — COMMENTED** [review]
The site looks great and you also did the strech goals which was cool. I liked how you displayed the amount of likes that you have made, it went well with the design. The code is easy to read and looks nice. You could divide the code into more components to make it easier to find each part of the website, but the functionality of the site is working as it should. Great work Josephine!

**Pull request closed without merging**